lib/action_subscriber/bunny/subscriber.rb

```diff
@@ -17,6 +17,9 @@ def auto_pop!
         times_to_pop = [::ActionSubscriber::Threadpool.ready_size, ::ActionSubscriber.config.times_to_pop].min
         times_to_pop.times do
           queues.each do |route, queue|
+            # Handle busy checks on a per threadpool basis
+            next if route.threadpool.busy_size >= route.threadpool.pool_size
```

Comment:
Scratch that. I was looking at the class method. But since there's a class method, that's all the more reason to add an instance method as well.